draft-ecahc-moderation.md

```diff
@@ -270,11 +270,14 @@ create throw-away personas for such denial-of-service behavior.)
   processes and actions. Should this document define some boundaries
   for what the moderator team can do?
 
-The moderator team SHOULD create and maintain a public mailing list
-for the community to discuss both the general moderation criteria and
-individual moderation decisions. To not distract from the
-topic-oriented discussion on other IETF lists, such meta-discussions
-SHOULD be actively redirected to the moderation discussion list.
+The moderator team is responsible to the IETF Chair.  The IETF Chair
+MAY create or designate a forum to facilitate discussion about
+moderation, and refer interested parties to that forum.  All actions
```

Comment:
Let's not make the chair create that, let's require the moderation team to create (and moderate, hah) that forum.

(Pet peeve of mine is the org piling more things on the IETF chair, see my chair-may-delegate draft.)

Reply:
That's not appropriate in this instance.  The community is directed to that forum by this document, and only the IESG should be telling the community where to go.  I realize that this is "chair dumping" but in this case, it's a very rare event.

Reply:
OK. Please rebase and then let's put this on the IESG, too.